qadi_simple.py

```
@@ -704,13 +787,17 @@ def is_similar(a: str, b: str, threshold: float = 0.85) -> bool:
                     print(f"• **Improvement:** {metrics.get('fitness_improvement_percent', 0):.1f}% fitness increase")
                     print(f"• **Evaluation:** {metrics.get('total_ideas_evaluated', 0)} total ideas tested")
                     
-                    # Show cache stats only in verbose mode
+                    # Show cache stats always in verbose mode
                     if verbose:
                         cache_stats = metrics.get("cache_stats")
-                        if cache_stats and cache_stats.get("hits", 0) > 0:
+                        if cache_stats:
                             print(f"\n💾 Cache Performance:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)